Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBProject.cs

```diff
@@ -202,5 +197,9 @@ public string ProjectDisplayName
                 }
             }
         }
+
+        private const string DllName = "vba6.dll";
```

Comment:
Two comments:

1) I prefer to keep externs separated from the implementation but that can be dealt with in a separate PR.

2) Are we sure VB6 actually uses this function from VB**A**6 DLL? I would have thought it'd be `msvbvm60.dll` or something similar.